crates/forge/src/scarb.rs

```diff
@@ -75,9 +75,10 @@ mod tests {
     use indoc::{formatdoc, indoc};
     use scarb_metadata::MetadataCommand;
     use std::str::FromStr;
+    use test_utils::tempdir_with_tool_versions;
 
     fn setup_package(package_name: &str) -> TempDir {
-        let temp = TempDir::new().unwrap();
+        let temp = tempdir_with_tool_versions().unwrap();
```

Comment:
I don't think we should use functions from the `test_utils` outside the tests and `benchmarks`
cc @cptartur 

maybe it's time to consider a better place for test_utils dir?

Reply:
If `tempdir_with_tool_versions` is used in normal code, it should be added to the `forge` crate somewhere. And then we can still use it in tests.

Reply:
but it shouldn't appear in the `normal` code. it is used inside some scarb tests

Reply:
Ah, right. That's test code. I don't think it's inherently wrong to use code from `test_utils` in normal package. I'd just make sure we only use `test_utils` as a dev-dependency.